recipes/dparse/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "dparse" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00a5fdfa900629e5159bf3600d44905b333f4059a3366f28e0dbd13eeab17b19
+
+build:
+  number: 0
```

This package doesn't seem to have any binaries, thus it would be nice to make it `noarch: python`.
```suggestion
  number: 0
  noarch: python
```

---

Thank you for the direction!